src/app/App.tsx

```diff
@@ -38,30 +39,35 @@ const useInitApp = () => {
   return { isLoading: !initialized };
 };
 
-export default function App() {
+const App: FC = () => {
   const { isLoading } = useInitApp();
-
+  const location = useLocation();
+  const fallbackClasses =
```

Comment:
I don't think this is the right way to do this, we should probably add a util hook that can be called from within a page component to disable the chrome

Reply:
But I agree fallback should have the chrome disabled